web/js/components/util/switch.js

```diff
@@ -59,11 +70,11 @@ const Switch = (props) => {
         {tooltip
           && (
             <>
-              <FontAwesomeIcon icon="info-circle" id="switch-tooltip" />
+              <FontAwesomeIcon icon="info-circle" id={`${id}-switch-tooltip`} tabIndex="-1" />
               <Tooltip
                 placement="right"
                 isOpen={tooltipOpen}
-                target="switch-tooltip"
+                target={`${id}-switch-tooltip`}
                 toggle={() => { toggleTooltip(!tooltipOpen); }}
```

Comment:
I see a delayed tooltip display issue where it starts to render behind the filter modal as if the z-index transforms (probably not what's happening, but looks that way). Adding the `fade` false fixes this for me.
```suggestion
                toggle={() => { toggleTooltip(!tooltipOpen); }}
                fade={false}
```